lib/verify.js

```diff
@@ -25,10 +25,14 @@ function verifySignature(req, opts, cb) {
 	try {
 		// parse the request
 		var parsed = httpSignature.parseRequest(req, opts);
+
+		//Use different verification methods depending if the algorithm is HMAC or not
+		var is_hmac = parsed.algorithm.toLowerCase().indexOf('hmac') === 0;
```

Comment:
@aisipos Just have a few code-style nit-picks. Can you switch to `lowerCamelCase` (i.e., `isHmac`)? Also, it's been a long time since I've worked with `http-signature`... is `parsed.algorithm` always guaranteed to be provided? And what are the possible values?